GliaWidgets/Sources/CoreSDKClient/CoreSDKClient.Mock.swift

```diff
@@ -39,7 +39,8 @@ extension CoreSdkClient {
         createLogger: { _ in Logger.mock },
         getCameraDeviceManageable: { .mock },
         subscribeForQueuesUpdates: { _, _ in UUID().uuidString },
-        unsubscribeFromUpdates: { _, _ in }
+        unsubscribeFromUpdates: { _, _ in },
+        subscribeForUnreadSCMessageCount: { _ in UUID().uuidString }
```

Comment:
Since `UUID` is the dependency itself,  `UUID.mock.uuidString` has to be used. 
Also this change has to be done to `subscribeForQueuesUpdates` above.
```suggestion
        subscribeForUnreadSCMessageCount: { _ in UUID.mock.uuidString }
```